### PR TITLE
Add states to FSDrift

### PR DESCRIPTION
**Pull request by jtaleric**

### CODE REVIEW

**Comment by bengland2** [comment]
ok, so there are 2 states, Starting and Running.   I guess I could have done that, but I don't understand what it gets you yet.  It looks like you want the long-running tasks to each be in a separate state, and the quick initialization stuff all gets shoved into Starting state.   Is that it?  Go ahead and merge, I'm ok with it.

**Comment by dry923** [comment]
LGTM however wondering where the minishift test is

File: roles/fs-drift/tasks/main.yml [hunk 1]
```diff
@@ -1,90 +1,151 @@
...
+      namespace: "{{ operator_namespace }}"
+      status:
+        state: Running
+        complete: false
+
+  when: resource_state.resources[0].status.state == "Starting"
+
+- block:
+
+  - name: Waiting for pods to complete....
```
[the remark this answers is not in the document]
Comment (jtaleric) [reply roles/fs-drift/tasks/main.yml]: It is actually accurate, as we are querying the state, and the next task (which isn't named) is checking the status.

**Pull request merged**